blok1.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace w02
+{
+    class blok
+    {
+        public int iloscPieter;
+        public bool senior;
+        public void CzyWinda()
+        {
+            this.iloscPieter = iloscPieter;
+            this.senior = senior;

```

Comment:
tu coś nie do końca wygląda dobrze. Skąd są wartości iloscPieter i senior? Nie są podane w metodzie po co została wykonana ta operacja? Proszę się zastanowić czy to cokolwiek robi